src/box/xrow_update_field.c

```diff
@@ -296,102 +324,135 @@ xrow_update_read_arg_splice(struct xrow_update_op *op, const char **expr,
 
 /* {{{ do_op helpers. */
 
+/*
```

Comment:
Lets use `/**` opener for out-of-function comments. Same in the other places.

Reply:
Fixed.